src/Orleans/Streams/PersistentStreams/IStreamQueueBalancer.cs

```diff
@@ -8,27 +8,32 @@ namespace Orleans.Streams
     /// The stream queue balancer is responsible for load balancing queues across all other related queue balancers.  It
     /// notifies any listeners (<code>IStreamQueueBalanceListener</code>) of changes to the distribution of queues.
     /// </summary>
-    internal interface IStreamQueueBalancer
+    public interface IStreamQueueBalancer
     {
         /// <summary>
+        /// Async Initialize logic
```

Comment:
hmm, I'm not sure I follow. What else instance will this be initializing? it is initializing itself. So "this instance" is a bit redundant to me